### ISSUE TITLE
off-line linear evaluation

### ISSUE THREAD

**Issue by allenhaozhu** [vturrisi/solo-learn#216]
I did an evaluation on the barlow-twin model (ResNet18 on CIFAR ) provided by your checkpoint. The result is 91.56 acc@1. So is there anything wrong？ I used 
` python3 ../../main_linear.py     --dataset cifar10     --backbone resnet18     --data_dir ./     --max_epochs 100     --gpus 0     --sync_batchnorm     --precision 16     --optimizer sgd     --scheduler step     --lr 0.1     --lr_decay_steps 60 80     --weight_decay 0     --batch_size 128     --num_workers 4     --name general-linear-eval     --pretrained_feature_extractor cifar/trained_models/barlow_twins/barlow-cifar10-otu5cw89-ep\=999.ckpt     --project self-supervised     --wandb`

**Comment by vturrisi** [comment]
Hey,
The numbers that we provide for cifar are from the online evaluation. In our experiments, these are usually a little higher than what you would get with the standard offline evaluation. Probably if you train for longer or tune the parameters, you will get the same performance.

**Comment by vturrisi** [comment]
I'm closing this, if you need anything else feel free to re-open.
